staging/src/k8s.io/apiserver/pkg/server/options/config/aes_config.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package aes transforms values for storage at rest using AES-GCM.
+package config
+
+import (
+	"crypto/aes"
+	"encoding/base64"
+	"fmt"
+
+	"k8s.io/apiserver/pkg/storage/value"
+	aestransformer "k8s.io/apiserver/pkg/storage/value/encrypt/aes"
+)
+
+type Key struct {
+	Name   string `json:"name"`
+	Secret string `json:"secret"`
+}
+
+// Config contains the API configuration for an AES transformer
+type AesConfig struct {
```

Comment:
style: we always capitalize leading acronyms/initialisms (`AESConfig`) in go code.  When it's private, you'd do `aesConfig`